hack/build-go.sh

```diff
@@ -7,6 +7,10 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 build_targets=("$@")
 platform="$(os::build::host_platform)"
 
+# Set build tags for these binaries
+readonly OS_GOFLAGS_TAGS="include_gcs include_oss"
+readonly OS_GOFLAGS_TAGS_$(os::build::platform_arch)="gssapi"
```

Comment:
@smarterclayton why did these get put into `build-cross` only in current `master`?